### ISSUE TITLE
v2 on Windows: “os error 3” migrating resource file

### ISSUE THREAD

**Issue by G-F-D** [cpsdqs/cohost-dl#29]
```
[INFO  cohost_dl::dl] migrating resource file rc\external\img.itch.zone\aW1nLzEzNzc5NDI0LnBuZw==\315x250%23c\zIazI3.png -> rc\external\img.itch.zone\aW1nLzEzNzc5NDI0LnBuZw==\315x250-23c\zIazI3.png
[ERROR cohost_dl::dl] The system cannot find the path specified. (os error 3)
```

Running Windows 10. Not sure why `%23` is becoming `-23` instead of `#` or why this is failing. That file _does_ exist at that exact location. Trying to do the rename manually or removing the file/directory entirely do not appease it.

**Comment by c-probably-archived20260718** [comment]
maybe I need to sacrifice more goats

**Comment by c-probably-archived20260718** [comment]
well, I looked it up, and apparently % is a character used for path variables on Windows, so that must be why it’s in the exclusion list. The actual bug here fortunately had nothing to do with Windows, it’s just that I forgot to create the directories…

fixed in ac0b256
